lib/matplotlib/colors.py

```diff
@@ -703,12 +726,36 @@ def _repr_html_(self):
         """Generate an HTML representation of the Colormap."""
         png_bytes = self._repr_png_()
         png_base64 = base64.b64encode(png_bytes).decode('ascii')
-        return ('<strong>' + self.name + '</strong>' +
-                '<img ' +
+        def color_block(color):
+            hex_color = to_hex(color, keep_alpha=True)
+            return ('<div title="' + hex_color + '" ' +
+                    'style="display: inline-block; ' +
+                    'width: 1em; height: 1em; ' +
+                    'margin: 0; ' +
+                    'vertical-align: middle; ' +
+                    'border: 1px solid #555; ' +
+                    'background-color: ' + hex_color + ';">' +
+                    '</div>')
```

Comment:
Use f-strings and implicit concatenation for better readability:
```suggestion
            return (f'<div title="{hex_color}" style="'
                    'display: inline-block; '
                    'width: 1em; height: 1em; '
                    'margin: 0; '
                    'vertical-align: middle; '
                    'border: 1px solid #555; '
                    f'background-color: {hex_color};"></div>')
```